src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/SecretBuildWrapper/config.jelly

```diff
@@ -29,4 +29,9 @@ THE SOFTWARE.
             <f:repeatableHeteroProperty field="bindings" hasHeader="true"/>
         </f:block>
     </f:section>
+    <f:advanced>
+    	<f:entry title="Show bindings" field="showBindings" description="If checked, all secret variable bindings will be clear-text.">
```

Comment:
I though that it would be really useful for debugging especially, if you want to see if there are any errors related to your password having a typo or similar or a path that is not accessible at all.

The old behaviour would is to show all bindings, so that option would show the same behaviour as before.